src/coreclr/vm/excep.cpp

```diff
@@ -4462,19 +4461,6 @@ LONG InternalUnhandledExceptionFilter_Worker(
 
         if (pParam->pThread != NULL)
         {
-
-            // In CoreCLR, we can be asked to not let an exception go unhandled on managed threads.
-            // If the exception reaches the top of the thread's stack, we simply deliver AppDomain's UnhandledException event and
-            // return back to the filter, instead of letting the process terminate because of unhandled exception.
-
-            // This code must only be exercised when running as a normal filter; returning
-            // EXCEPTION_EXECUTE_HANDLER is not valid if this code is being invoked from
-            // the UEF.
-            // Fortunately, we should never get into this case, since the thread flag about
-            // ignoring unhandled exceptions cannot be set on the default domain.
-
-            BOOL fIsProcessTerminating = !(AppDomain::GetCurrentDomain()->IgnoreUnhandledExceptions());
```

Comment:
This functionality is superseded by #101560

Reply:
> // Fortunately, we should never get into this case, since the thread flag about
            // ignoring unhandled exceptions cannot be set on the default domain.

Does this mean the feature could not be enabled on CoreCLR?

Reply:
This comment is wrong. It could be enabled on CoreCLR if somebody uses untested and undocumented `APPDOMAIN_IGNORE_UNHANDLED_EXCEPTIONS` flag on the unsupported legacy CoreCLR hosting API.